Cargo.toml

```diff
@@ -22,10 +22,13 @@ rust-version = "1.60.0"
 [features]
 default = ["tracing-log", "metrics"]
 # Enables support for exporting OpenTelemetry metrics
-metrics = ["opentelemetry/metrics", "smallvec"]
+metrics = ["opentelemetry/metrics","opentelemetry_sdk/metrics", "smallvec"]
 
 [dependencies]
 opentelemetry = { version = "0.20.0", default-features = false, features = ["trace"] }
+# TODO: Use blanket implementation in the future over tracer so that
```

Comment:
Hm I'm not sure that you can implement `PreSampledTracer` in a generic way as it needs to call methods that are not available on the `Tracer` trait. Maybe just remove this TODO for now as the other SDK may have to implement it on their end. We could have the sdk be an optional dependency though potentially to suggest this path.

Reply:
🆗